package.json

```diff
@@ -25,6 +25,8 @@
     "babel-eslint": "^8.2.1",
     "eslint": "^4.17.0",
     "nsp": "^3.1.0",
+    "fs": "0.0.1-security",
```

Comment:
forgot this one - 'fs' is an inbuilt node library, no need to add it to package.json